app/models/news.rb

```diff
@@ -0,0 +1,4 @@
+class News < ApplicationRecord
+  validates :title, presence: true, length: { minimum: 3 }
```

Comment:
add relation with user model

Reply:
@okohuch added, now trying to create specs